libs/vm_monitor/src/cgroup.rs

```diff
@@ -232,6 +249,23 @@ impl MemoryStatus {
 
         DS(slice)
     }
+
+    /// Check if the other memory status is a close or similar result.
+    /// Returns true if the larger value is not larger than the smaller value
+    /// by 1/8 of the smaller value, and within 128MiB.
+    fn status_is_close_similar(&self, other: &MemoryStatus) -> bool {
```

Comment:
Could rename to:

- `status_is_close_or_similar`
- `status_has_meaningfully_changed`

Food for thought.